contents/blog/how-to-fix-your-growth-team.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Why your growth team isn't winning (and how to fix it)
+date: 2023-09-14
+author: ["lior-neu-ner"]
+showTitle: true
+rootpage: /blog
+sidebar: Blog
+hideAnchor: true
+featuredImage: ../images/blog/athlete-hog.jpeg
+featuredImageType: full
+category: Product growth
+tags: 
+  - Guides
+  - Product metrics
+  - Product engineers
+---
+
+There are tell-tale signs when a growth team isn't succeeding: Stuck metrics, missed KPIs, and burnt-out engineers. Everyone is frustrated, but how do you fix it?
+
+[Alexey Komissarouk](https://alexeymk.com/), former head of growth engineering at [MasterClass](https://www.masterclass.com/), has seen it all. He grew Masterclass xyz... 
+
+In this post, Alexey shares his advice on how to get your growth team winning again.
+
+TLDR; there are 3 areas to investigate:
+
+1. Your team's velocity.
+2. Your experiment win rate.
+3. Your experiment win size.
+
+## Problem 1: Your team is moving too slowly
+
+Your PM complains that the team is moving too slowly. The engineering team says there is too much technical debt. Sound familiar? This is caused by engineers having the wrong mindset.
+
+Slow velocity is usually a result of code quality being too high. "Engineers hate doing growth. They want to implement elegant solutions to complex problems. Growth engineering is the opposite of that", says Alexey.
+
+### How to fix it
+
+First, sit with the engineering team and prioritize your technical debt. Figure out what is *actually* slowing you down (or likely to) and by how much. Once you have a ranked list, assign 10%-20% of engineering time to clean this up.
```

Comment:
Is this something growth should do or the rest of the org? Sounds a bit big for growth to be responsible for.

Reply:
Going to require a lot of buy-in up the change to restructure what eng works on.

Reply:
In the context of this blog, it's the growth engineering team. I dropped the "growth" part to avoid overly repeating myself. 

I can add it/rephrase if you feel its still unclear?